src/components/review/data-list/table.tsx

```diff
@@ -0,0 +1,115 @@
+'use client';
+
+import React from 'react';
+import { useRouter } from 'next/navigation';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@/components/ui/table';
+import { Button } from '@/components/ui/button';
+import { Badge } from '@/components/ui/badge';
+import { ArrowRight } from 'lucide-react';
+import type { MonthlySummary } from '@/lib/review/types';
+
+interface ReviewTableProps {
+  summaries: MonthlySummary[];
+  onReview: (district: string, month: string) => void;
+}
+
+export function ReviewTable({ summaries, onReview }: ReviewTableProps) {
+  const router = useRouter();
+
+  const handleReview = (district: string, month: string) => {
+    // Navigate to detailed review page
+    router.push(`/review/${district}/${month}`);
+  };
+
+  if (summaries.length === 0) {
+    return (
+      <div className="py-12 text-center">
+        <p className="text-muted-foreground">
+          No review data found for the selected filters.
+        </p>
+      </div>
+    );
+  }
+
+  return (
+    <div className="rounded-md border">
+      <Table>
+        <TableHeader>
+          <TableRow className="bg-muted/50">
+            <TableHead className="font-semibold">District / Month</TableHead>
+            <TableHead className="font-semibold">Status</TableHead>
+            <TableHead className="font-semibold">KPIs</TableHead>
+            <TableHead className="font-semibold">Last Updated</TableHead>
```

Comment:
Some of these columns can be removed for now. Maybe we can have columns for the district, month, and review action